CHANGES.rst

```diff
@@ -17,6 +17,8 @@ New Features
 Cubeviz
 ^^^^^^^
 
+- Enhancements for the cube sonification plugin. [#3387]
```

Comment:
are you expecting to combine this with other PRs?  Should we add more details or is this intentionally vague?

Reply:
Yes I'm expecting to combine it with the change message in #3377

Reply:
ok, feel free to merge then and do so!